gradle.properties

```diff
@@ -7,6 +7,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 org.gradle.jvmargs=-Xmx1536m
+DecentWork_GoogleClientID= "639739154219-u3vfm29sduc1pk391a153itafjgiuomn.apps.googleusercontent.com"
```

Comment:
Nani? This app key should go out of repository. You should have .gradle directory in your home directory and make there file gradle.properties. Then move this property there.